src/goose/cli/prompt/create.py

```diff
@@ -6,10 +6,10 @@
 
 from goose.cli.prompt.completer import GoosePromptCompleter
 from goose.cli.prompt.lexer import PromptLexer
-from goose.command import get_commands
+from goose.command.base import Command
 
 
-def create_prompt() -> PromptSession:
+def create_prompt(commands: dict[str, Command]) -> PromptSession:
```

Comment:
I know this is an old function but can we get a docstring since it's part of your PR 🙏

Reply:
for sure, good call!